apps/judicial-system/backend/src/app/modules/notification/notification.service.ts

```diff
@@ -69,6 +69,14 @@ export class NotificationService {
           ]
         } else {
           messages = [this.getNotificationMessage(type, user, theCase)]
+          theCase.defendants?.forEach((defendant) => {
+            messages.push({
+              type: MessageType.DELIVERY_TO_POLICE_SUBPOENA,
```

Comment:
As discussed this belongs else where. Perhaps it would make sense to add a TODO comment here to remind us.